### ISSUE TITLE
Mock PA2 Sample

### ISSUE THREAD

**Issue by ZYN2122** [nus-cs2030/2223-s1#245]
# Summary
Put a summary of the problem here
Hi, just wondering, will the Mock PA2 sample solution be uploaded for us to see (like Mock PA1)? 
Thanks!
## Description 
Describe the problem and any details if necessary

## Idea s
Describe the approach used (if applicable)

## Screenshots (if any): 
Insert Images here if necessary

**Comment by glendon29** [comment]
PA2 is a graded assignment and therefore no solutions will be uploaded based on what my TA said

**Comment by mohddanesh** [comment]
i think not

**Comment by Wenbo-J** [comment]
Nope, this is due to the Mock PA2 being a part of the graded labs (It's Lab #7)

**Comment by ZYN2122** [comment]
Ah okay thanks!
